File: generated-code-groupA-updated/generated-code-groupA-updated/chatgpt/java-chatgpt-A/Submit_customer_feedback_020_Vulnerable.java
import java.util.ArrayList;
import java.util.List;

public class Submit_customer_feedback_020_detelid {

    static class Feedback {
        String name;
        String email;
        int rating;
        String message;

        Feedback(String n, String e, int r, String m) {
            name = n;
            email = e;
            rating = r;
            message = m;
        }
    }

    static List<Feedback> feedbacks = new ArrayList<>();

    public static String submitFeedback(
            String name,
            String email,
            int rating,
            String message) {

        feedbacks.add(new Feedback(name, email, rating, message));

        return "Feedback submitted successfully.";
    }
}

File: generated-code-groupA-updated/generated-code-groupA-updated/chatgpt/python-chatgpt-A/Submit_customer_feedback_020_Vulnerable.py
feedbacks = []


def submit_customer_feedback(name, email, rating, message):
    """
    Submits customer feedback.
    """

    feedbacks.append({
        "name": name,
        "email": email,
        "rating": rating,
        "message": message
    })

    return "Feedback submitted successfully."


if __name__ == "__main__":
    print(submit_customer_feedback(
        "Ahmed",
        "ahmed@example.com",
        5,
        "Great service!"
    ))